docs/documentation/development/bindings/xml-reference.md

```diff
@@ -101,6 +101,16 @@ The following HTML tags are allowed -: ```<b>, <br>, <em>, <h1>, <h2>, <h3>, <h4
   <tr><td>criteria.name</td><td>The name of the context related filter.</td></tr>  
 </table>
 
+Context is used to render different kind of input widgets on front-end. Following contexts are currently supported in PaperUI.
```

Comment:
Context is not primarily about UIs, but to provide some "semantics" for the field.